### ISSUE TITLE
Artificial surprises through endpoint from wbor.org, instead of local ones

### ISSUE THREAD

**Issue by CaspianA1** [WBOR-91-1-FM/wbor-studio-dashboard#30]

**Comment by mdrxy** [comment]
This would likely need to be a local endpoint, e.g. `wbor-dashboard.bowdoin.edu/surprise?{ASSET_NAME}`

**Comment by CaspianA1** [comment]
So it would run on the bowdoin network but not beyond that?

**Comment by mdrxy** [comment]
Yep since we don't have a path through the campus firewall, and the next best option is continually polling our remote VPS.
